### DISCUSSION TITLE
Welcome to I, Voyager!

### DISCUSSION THREAD

**Discussion by charliewhitfield** [ivoyager/forum#1] [Announcements]
This forum is for discussion of [I, Voyager](https://ivoyager.dev/) and all things related, which could include astronomy, space exploration, space-and science-based games, game development (especially the [Godot Engine](https://godotengine.org/)), educational software, web design, programming, mathematics, science, science fiction, futurism, and probably other topics I haven't thought of.

Feel free to introduce yourself in this thread.

Happy Voyaging!

**Comment by mbgaron** [comment]
Hi everyone! I'm here from the old forums. I had found iVoyager while searching for a Godot-based planetarium to form the basis of a "hard sci fi" space game, set in the world of the TTRPG Eclipse Phase. I dabble in art (2D and 3D) and music production. I look forward to seeing where iVoyager goes, and how I might be able to contribute!

**Reply by charliewhitfield** [reply]
I’ll help as much as I can on the I, Voyager side. We’ll be porting to Godot 4.x by the end of the year (I’m pretty sure). So there will be large API changes. But the basic architecture will be the same, so try modifying from the Project Template now to learn how things work.
